platform/qt/src/format_number.cpp

```diff
@@ -0,0 +1,34 @@
+#include <mbgl/util/platform.hpp>
+
+#include <QtQml/QtQml>
+
+namespace mbgl {
+namespace platform {
+
+std::string formatNumber(double number, const std::string& localeId, const std::string& currency,
```

Comment:
Could this be implemented with https://doc.qt.io/qt-5/qlocale.html ?

Reply:
Done